akka-docs/src/main/paradox/stream/operators/index-alphabetical.md

```diff
@@ -2,249 +2,265 @@
 
 | |Operator|Description|
 |--|--|--|
-|Source/Flow|@ref[actorRef](actorRef.md)|Materialize an `ActorRef`, sending messages to it will emit them on the stream. |
+|Source/Flow|@ref[actorRef](source-or-flow/actorRef.md)|Materialize an `ActorRef`, sending messages to it will emit them on the stream. |
```

Comment:
that's only Source :-)

Reply:
We can fix those tomorrow morning, filter our those which are only one of the things flow/source; have a good night today man :-)